Plugins/Flow.Launcher.Plugin.BrowserBookmark/FirefoxBookmarkLoader.cs

```diff
@@ -9,10 +9,10 @@ namespace Flow.Launcher.Plugin.BrowserBookmark
 {
     public class FirefoxBookmarkLoader : IBookmarkLoader
     {
-        private const string queryAllBookmarks = @"SELECT moz_places.url, moz_bookmarks.title
+        private const string QueryAllBookmarks = @"SELECT moz_places.url, moz_bookmarks.title
               FROM moz_places
               INNER JOIN moz_bookmarks ON (
-                moz_bookmarks.fk NOT NULL AND moz_bookmarks.fk = moz_places.id
+                moz_bookmarks.fk NOT NULL AND moz_bookmarks.title NOT NULL AND moz_bookmarks.fk = moz_places.id
```

Comment:
why is this condition added? 

also would this `moz_bookmarks.fk = moz_places.id` be an `==` instead?

Reply:
They are generally duplicate